CHANGELOG.md

```diff
@@ -75,6 +77,9 @@ be found.
 
 ### Networking
 
+- Fix panic if a node is forcibly removed from the cluster ([#21671](https://github.com/docker/docker/pull/21671))
+- Fix "error creating vxlan interface" when starting a container in a Swarm cluster ([#21671](https://github.com/docker/docker/pull/21671))
+- Fix `host` and `none` networks to disappear until restart when SIGHUP is used to reload cluster configuration ([#21671](https://github.com/docker/docker/pull/21671))
```

Comment:
I don't think this needs to be in there.
We didn't support reloading the cluster config in 1.10